.changeset/hot-geckos-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/modal': major
```

Comment:
Why major? Isn't this additional functionality? What functionality does this break?